docusaurus.config.js

```diff
@@ -86,6 +99,31 @@ const config = {
         },
       },
     ],
+    [
+      'docusaurus-plugin-remote-content',
+      {
+        name: 'vscode-extension',
+        sourceBaseUrl: `https://raw.githubusercontent.com/DevCycleHQ/vscode-extension/${VSCODE_EXTENSION_VERSION}`,
+        outDir: 'docs/integrations/vscode-extension',
+        documents: [
+          'README.md',
+          'DEVELOPMENT.md',
```

Comment:
we likely don't need the development instructions on the docs site

Reply:
good point, i'll remove it